contracts/evm/contracts/Stork.sol

```diff
@@ -67,6 +67,17 @@ abstract contract Stork is StorkGetters, StorkSetters, StorkVerify {
         return numericValue;
     }
 
+    function getTemporalNumericValueUnsafeV1(
```

Comment:
copied from `getTemporalNumericValueV1` but without the "stale value" check so we'll always return a latest value